src/pages/resultsView/settings/DriverAnnotationControls.tsx

```diff
@@ -107,6 +104,23 @@ export default class DriverAnnotationControls extends React.Component<
         }
     }
 
+    @autobind
+    private onCustomDriverTierCheckboxClick(
+        event: React.MouseEvent<HTMLInputElement>
+    ) {
+        let a = 1;
```

Comment:
is a used?

Reply:
@alisman No I have removed the line. I think that all e2e-tests that fail are not because of my code. Can we merge?